app/components/radio-button/template.hbs

```diff
@@ -0,0 +1 @@
+
```

Comment:
this should be empty

Reply:
can you explain why?

Reply:
The actual element is created by the component.js above, see `tagName` and `type`